src/components/ec/base/ucc_ec_base.h

```diff
@@ -63,8 +63,10 @@ enum ucc_ee_executor_params_field {
 };
 
 typedef enum ucc_ee_executor_task_type {
-    UCC_EE_EXECUTOR_TASK_TYPE_COPY   = UCC_BIT(0),
-    UCC_EE_EXECUTOR_TASK_TYPE_REDUCE = UCC_BIT(1),
+    UCC_EE_EXECUTOR_TASK_TYPE_COPY               = UCC_BIT(0),
+    UCC_EE_EXECUTOR_TASK_TYPE_REDUCE             = UCC_BIT(1),
+    UCC_EE_EXECUTOR_TASK_TYPE_REDUCE_MULTI       = UCC_BIT(2),
+    UCC_EE_EXECUTOR_TASK_TYPE_REDUCE_MULTI_ALPHA = UCC_BIT(3),
```

Comment:
@Sergei-Lebedev ALPHA?

Reply:
My comment is about naming - Why alpha?

Reply:
This is common for executor and regular reduce
UCC_EE_EXECUTOR_TASK_TYPE_REDUCE:                         C = A1 op A2
UCC_EE_EXECUTOR_TASK_TYPE_REDUCE_MULTI:              C = A1 op A2 op ... op AN
UCC_EE_EXECUTOR_TASK_TYPE_REDUCE_MULTI_ALPHA: C= alpha * (A1 op A2 op ... op AN)
C, A1, A2, AN - vectors
alpha - scalar